### PR TITLE
[TW-1539] Overview refresh Public API network

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Refreshed `public-api-network-overview.md`: updated the featured snippet and screenshot, and added descriptions about topics in the section
* Updated the screenshot and outdated details at `exploring-public-api-network.md`
* Updated other minor occurrences of outdated instructions

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/getting-started/first-steps/exploring-public-api-network.md [hunk 1]
```diff
@@ -35,18 +35,18 @@ contextual_links:
...
+* The **Search Postman** box at the top of the page searches all Postman teams, workspaces, collections, APIs, and Flows by default. You can also change the scope to **Public API Network** to limit your search to the Public API Network.
+* **Spotlight** highlights the latest exciting, useful, and noteworthy contributions to the Public API Network, handpicked by the experts at Postman.
+* **Trending APIs** shows elements that are the most popular in the Public API Network, based on the number of people viewing or creating forks of them.
+* **Categories** showcases popular API categories in the Public API Network: [Artificial Intelligence](https://www.postman.com/category/artificial-intelligence), [Communication](https://www.postman.com/category/communication), [Data Analytics](https://www.postman.com/category/data-analytics), [Developer Productivity](https://www.postman.com/category/developer-productivity), [DevOps](https://www.postman.com/category/devops), [Financial Services](https://www.postman.com/category/financial-services), [Payments](https://www.postman.com/category/payments), and more.
```
Comment (akinard-postman) [inline src/pages/docs/getting-started/first-steps/exploring-public-api-network.md:47]: Is it possibly worth removing the specific categories in the event that this gets added to/revised in the UI and no one catches it?

**Review by akinard-postman — APPROVED** [review]
Just one comment but looks good to me! 👍

File: src/pages/docs/getting-started/first-steps/exploring-public-api-network.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/getting-started/first-steps/exploring-public-api-network.md:47]: Might be worth leaving this for SEO, actually.

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/getting-started/first-steps/exploring-public-api-network.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/first-steps/exploring-public-api-network.md:47]: OK. I can leave the content as is.

**Pull request merged**